### PR TITLE
EVG-18175: Upgrade Tooltip and Popover

### PR DESCRIPTION
**Pull request by sophstad**
EVG-18175

### Description

- No API changes. Visually looks pretty much the same too!

### Screenshots

image
image

### Testing
- Unit test flakiness tracked in EVG-18333

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,7 +79,7 @@
     "@leafygreen-ui/menu": "18.0.0",
     "@leafygreen-ui/modal": "6.1.2",
     "@leafygreen-ui/palette": "3.4.4",
-    "@leafygreen-ui/popover": "7.2.2",
+    "@leafygreen-ui/popover": "11.0.0",
     "@leafygreen-ui/radio-box-group": "12.0.0",
     "@leafygreen-ui/radio-group": "10.0.0",
```
Comment (minnakt) [inline package.json:84]: Dark purple doesn't look too good inside the tooltip, sadly... we can revisit it later 😔
Screen Shot 2022-11-18 at 4 57 16 PM

Comment (minnakt) [inline package.json:84]: This one random tooltip is in `darkMode`, would you mind taking the prop off for it 😅

Screen Shot 2022-11-18 at 5 01 49 PM

File: package.json [hunk 2]
```diff
@@ -79,7 +79,7 @@
...
     "@leafygreen-ui/segmented-control": "5.0.0",
```
Comment (minnakt) [inline package.json:85]: (writing this mostly as a reminder to myself) The new tooltips make hovering on the waterfall icons pretty funky, it's possible we need to reduce the delay

**Review by minnakt — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (khelif96) [reply package.json:85]: Can we ticket this? Cailin also mentioned that highlighting tooltips on the waterfall is too slow for her liking. So if it's even slower we should address it before releasing it.

File: package.json (hunk 1 above)
Comment (sophstad) [reply package.json:84]: Good catch—a lot of these icons actually get pretty messed up with the dark background:
image
I'm going to apply dark mode here as a workaround
image

File: package.json (hunk 2 above)
Comment (sophstad) [reply package.json:85]: done EVG-18374

**Pull request merged**